digitsGroup_test.go

```diff
@@ -0,0 +1,76 @@
+package digits
+
+type Stimulus struct {
+	value               string
+	groupingCharRune    rune
+	groupingCharStr     string
+	ifSigFig            bool
+	fractionalPrecision Decimals
+}
+type Output struct {
```

Comment:
Do not make this struct public within the `digits` package.  This will be included in the library's binary.

Reply:
3cc05b7